docs/features/star-search.md

```diff
@@ -23,6 +28,30 @@ keywords:
 - **Hidden Experts**: Discover untapped talent in the ecosystem.
 
 ![StarSearch](../../static/img/star-search.png)
-## How to Access StarSearch
 
-Currently, StarSearch is in development, but you can [sign up for the waitlist now](https://app.opensauced.pizza/star-search/waitlist).
\ No newline at end of file
+### Queries Suggestions
+
+#### **Contributor Activities**
+    - Show me the contributions of \{username}
+    - What \{technology} contributions has \{username} made recently?
+    - What issues has \{username} commented on?
+- **Work-Based Connections**: 
+    - Who has contributed to both project A and project B?
+    - Who are the contributors to both \{technology} and \{technology}?
+- **Hidden Experts**: 
+    - Who are the experts in \{technology}?
+    - Who are the most prevalent contributors to the \{technology} ecosystem?
```

Comment:
There are a couple of things here:

- There should be a space after the heading.
- Because there are "Work-Based Connections" and "Hidden Experts", can we group the first three lists in something like "Contribution Activities" or "General Activities"? 

```suggestion
#### **Contributor Activities**

- **Contributions Activities**:
  - Show me the contributions of \{username}
  - What \{technology} contributions has \{username} made recently?
  - What issues has \{username} commented on?
- **Work-Based Connections**:
  - Who has contributed to both project A and project B?
  - Who are the contributors to both \{technology} and \{technology}?
- **Hidden Experts**:
  - Who are the experts in \{technology}?
  - Who are the most prevalent contributors to the \{technology} ecosystem?
```